Sources/Extensions/UIDeviceExtensions.swift

```diff
@@ -30,7 +30,11 @@ extension UIDevice {
     var freeDiskSpaceInBytes: ByteCountFormatter.Units.Bytes {
         if #available(iOS 11.0, *) {
             if let space = try? URL(fileURLWithPath: NSHomeDirectory() as String).resourceValues(forKeys: [URLResourceKey.volumeAvailableCapacityForImportantUsageKey]).volumeAvailableCapacityForImportantUsage {
-                return space
+                #if swift(>=5)
+                    return space
+                #else
+                    return space ?? 0
```

Comment:
Would this mean that the feature doesn't work on Swift 5.0 and lower? If so, I rather fix this by adding actual support instead of just making it work!

Reply:
I added this since on swift 4/4.2 `volumeAvailableCapacityForImportantUsage` is still an optional even after the `if let` statement and on swift 5 you can unwrap the value by only using on `if let` statement

Reply:
Gotya, makes sense!